rfcs/text/0001-rfc-process.md

```diff
@@ -70,103 +84,111 @@ Where necessary, existing RFCs can be revised using the same process (a proposed
 
 ## Detailed Design
 
-### Proposing RFCs
+### Drafting RFCs
 
 #### **Authors**:
 
-*Before proposing, always engage the DCP community to build consensus and assess whether your proposal sparks interest and addresses a need. You may even discover potential collaborators.*
+There are three types of RFCs that may be proposed: ***Software***, ***Governance***, and ***Informational***. Software RFCs are expected to be linked to an Objective that is prioritized in the DCP Product Roadmap, tracked by linking directly to an existing ticket in the DCP Zenhub board (usually labeled as a Spike). Informational and Governance RFCs need not directly reference an open ticket in the DCP Zenhub board.
 
-  - Fork the HumanCellAtlas dcp-community repository
-  - Copy `rfcs/rfc-template.md` to `rfcs/text/0000-my-feature.md`
-  *- where "my-feature" is descriptive. Don't assign an RFC number yet.*
-  
-  - Fill in the RFC with attention to detail
+To inform the DCP Community about the development of an RFC, with the intention of forming collaborations between people thinking about similar problems, it is RECOMMENDED that a draft of the RFC be published as soon as the Problem Statement and Authors have been established. The steps to begin are as follows:
 
-  - Submit a pull request.
-  
-      For software RFCs:
-      - Add the "rfc-community-review" and the appropriate software project name _(such as "Data Store")_ labels. When the RFC impacts multiple DCP software projects, then the "Architecture" label **MUST** be the project name. *If uncertain about the appropriate project name, then **_Ask a PM_** on the HCA **#dcp-project-mgmt** slack channel*
+   1. Fork the HumanCellAtlas `dcp-community` repository
+   2. Copy `rfcs/rfc-template.md` to `rfcs/text/0000-my-feature.md`. Make `my-feature` descriptive but do not assign an RFC number yet.
+   3. Fill out the Problem Statement and Author(s) sections.
+   4. Push the changes to your branch and create a _Draft_ PR.
```

Comment:
I would not assume that everyone knows about Draft PR(s). I suggest adding a reference, changing `_Draft_ PR` to `[_Draft PR_](https://github.blog/2019-02-14-introducing-draft-pull-requests/)`